File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    let alphabet = "abcdefghijklmnopqrstuvwxyz";
    let NewSentence=[...new Set(sentence)].sort().join("");
    return NewSentence.includes(alphabet);
};

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cs
public class Solution {
    public bool CheckIfPangram(string sentence) {
        string alphabet = "abcdefghijklmnopqrstuvwxyz";
        string NewSentence= new string(sentence.Distinct().OrderBy(c => c).ToArray());
        return NewSentence.Contains(alphabet);
    }
}